ENG-5418: Create Collision Models in Forge that Exceed those in the Robot Controller

AliZamaniGithub opened the pull request:
This modifies the current collision models in forge by making them bigger than collision models in the robot controller such that the FSU doesn't get triggered.

[Jira Card](https://pathrobotics.atlassian.net/browse/ENG-5418)

Doomerdinger commented on motoman_ma1440_d500_moveit_config/launch/move_group.launch:
```diff
@@ -19,7 +19,7 @@
   <arg name="jiggle_fraction" default="0.05" />
   <arg name="publish_monitored_planning_scene" default="true"/>
 
-  <arg name="capabilities" default=""/>
```
It may be better to pass in this argument from the parent file that we use (ma1440_d500_moveit.launch) instead of changing the default value here.
Either way, the change should be mimicked in the ma1440_mh185 moveit package.

daboytim replied:
Made the change

AliZamaniGithub committed:
ENG-5418: Create Collision Models in Forge that Exceed those in the Robot Controller

This modifies the current collision models in forge by making them bigger than collision models in the robot controller such that the FSU doesn't get triggered.
